monkey/monkey_island/cc/resources/telemetry_feed.py

```diff
@@ -109,8 +109,17 @@ def get_post_breach_telem_brief(telem):
         )
 
     @staticmethod
-    def should_show_brief(telem):
-        return telem["telem_category"] in TELEM_PROCESS_DICT
+    def should_show_brief(telem) -> bool:
+        should_process_telem = SHOULD_PROCESS_TELEM.get(telem["telem_category"], lambda _: True)
+        return telem["telem_category"] in TELEM_PROCESS_DICT and should_process_telem(telem)
+
+    @staticmethod
+    def should_process_scan_telem(telem) -> bool:
+        return TelemetryFeed._machine_responded(telem["data"]["machine"])
+
+    @staticmethod
+    def _machine_responded(machine) -> bool:
+        return machine["icmp"] and len(machine["services"].keys()) > 0
```

Comment:
Can't we do
```
machine = telem["data"]["machine"]
if machine["icmp"] and len(machine["services"].keys()) > 0:
    return None
else:
    return "Monkey discovered machine %s." % telem["data"]["machine"]["ip_addr"]
```
In `cc.resources.telemetry_feed.TelemetryFeed.get_scan_telem_brief`?

Reply:
No, this happens if simply `None` is returned:
![image](https://user-images.githubusercontent.com/44770317/164441474-307ef921-6a61-40a2-905f-883452cc8967.png)